src/app/components/Btn.js

```diff
@@ -0,0 +1,23 @@
+export const Btn = (
+  btnObj = {id: undefined, btnText: '', class: [], onClickFn: undefined, icon: ''},
```

Comment:
class powinno mieć nazwę classes albo classList skoro to array :)

Reply:
Dzięki za uwagę! Nie pomyślałem o tym